test/test_errors.py

```diff
@@ -0,0 +1,18 @@
+from pymongo.errors import (AutoReconnect,
+                            NotMasterError,
+                            OperationFailure)
+from test import (client_context,
+                  client_knobs,
+                  PyMongoTestCase,
+                  sanitize_cmd,
+                  unittest)
+
+class TestErrors(PyMongoTestCase):
+    def test_not_master_error(self):
+        exc = NotMasterError("not master test", "details")
+        self.assertIn("full error", str(exc))
+        try:
+            raise exc
+        except NotMasterError:
+            import traceback
+            self.assertIn("full error", traceback.format_exc())
```

Comment:
Can you add the `if __name__ == "__main__":` logic at the bottom of this file? https://github.com/mongodb/mongo-python-driver/blob/4760d0781505c0f13d5d1bc061ff89075927eb2f/test/test_bson.py#L1088-L1089

Reply:
Done.